packages/themes/itzbund/src/components/pagination.scss

```diff
@@ -6,6 +6,19 @@
 		font-family: var(--font-family-sans);
 	}
 
+	.kol-pagination {
+		&__button {
+			&--selected .button-inner {
```

Comment:
Bitte analog zu meinem Beispiel im Default-Theme die Styles in den button-inner Code-Block verschieben.